MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -6046,9 +6048,9 @@ private void rmvwht()
         {
             var stringPointer = GetParameterPointer(0);
 
-            var stringToParse = Encoding.ASCII.GetString(Module.Memory.GetString(stringPointer, true));
+            var stringToParse = Encoding.ASCII.GetString(Module.Memory.GetString(stringPointer));
 
-            var parsedString = stringToParse.Trim() + '\0';
+            var parsedString = string.Concat(stringToParse.Where(c => !char.IsWhiteSpace(c)));
```

Comment:
string.Concat is confusing me since there's only one argument.

Will `new string(input.ToCharArray().Where(c => !char.IsWhiteSpace(c)))` work?

Reply:
that won't work because an Array works on bytes, and IsWhiteSpace only works on chars

I call the Where() LINQ method on the source string. That method filters the string's characters. Inside its parentheses we specify that every character c should not (!) be a whitespace character (IsWhiteSpace()). The result of that filter is a new series of characters. We join those together in a string again with the String.Concat() method.